### DISCUSSION TITLE
Installation stuck on Updating Container OS

### DISCUSSION THREAD

**Discussion by hesiodes** [tteck/Proxmox#3217] [Help Needed] [answered]
Hello,

This is my first time starting a discussion, and I would like to begin by thanking the developers for their excellent work. I used the platform several months ago, and as a true beginner, I found it very helpful.

Today, however, I tried multiple times to install Docker LXC or Frigate LXC, but the installation always gets stuck at the "Updating Container OS" step. Is there a specific reason for this? I noticed that significant changes are planned for July; have you started implementing any internal changes already?

As I mentioned, I am a complete beginner and not a native English speaker, so please be kind in your response! :)

**Comment by Mellowlynx** [comment]
Updating the OS can take some time, depending on your internet speed.
Just be sure to not move away/close the console and let it run, it should complete if it has an IP and internet connection.

**Reply by hesiodes** [reply]
Thank you for your response. I will test your solution. 
For your information, I have monitored the incoming and outgoing bandwidth on Proxmox during my numerous attempts. Initially, the connection reaches up to 100 kb/s, but after about twenty minutes, it consistently drops to levels between 1 and 2 kb/s.

**Reply by tteck** [reply]
Firewall?

**Comment by hesiodes** [comment, marked as answer]
The issue was due to the selection of the Debian version. I had chosen version 12 because I already had it on my Proxmox. By selecting the default version (v11) and downloading it during the script execution, the LXC installation went smoothly. Thank you for your help! And once again, thanks for your work!
